src/test/java/school/hei/haapi/integration/TeacherIT.java

```diff
@@ -168,24 +170,27 @@ void manager_write_update_rollback_on_event_error() throws ApiException {
   @Test
   void manager_write_create_ok() throws ApiException {
     ApiClient manager1Client = anApiClient(MANAGER1_TOKEN);
-    Teacher toCreate = someCreatableTeacher();
+    CrupdateTeacher toCreate = someCreatableTeacher();
+    Teacher expected = expectedSomeTeacherCreated();
```

Comment:
if not reused in other classes then put it in this class as private instead of TestUtils

Reply:
also, rename to expectedCreatedTeacher